app/src/main/res/layout/count_in_activity.xml

```diff
@@ -25,116 +24,179 @@
         android:layout_height="wrap_content"
         android:hint="Coin Total"
         app:layout_constraintBottom_toTopOf="@+id/dollar_view"
-        app:layout_constraintEnd_toStartOf="@+id/twentites_count"
+        app:layout_constraintEnd_toStartOf="@+id/twenties_count"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/quarters_count" />
 
-    <EditText
+
+    <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/fives_count"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:ems="10"
         android:hint="@string/fives"
-        android:inputType="number"
-        android:textSize="16sp"
```

Comment:
Why remove textsize?

Reply:
You can add this to the TextInputEditText if you want

Reply:
"But now it will default to a reasonable size"
- Aaron Egger